react_app/src/components/ChallengeCard.js

```diff
@@ -11,43 +12,47 @@ import Chip from "@material-ui/core/Chip";
 import Avatar from "@material-ui/core/Avatar";
 import Divider from "@material-ui/core/Divider";
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    margin: theme.spacing(3)
-  },
-  valueChip: {
-    position: "absolute",
-    marginLeft: -theme.spacing(1.5),
-    marginTop: -theme.spacing(1.5)
-  },
-  alignLeft: {
-    textAlign: "left"
+import "../styles/ChallengeCard.css";
+
+class ChallengeCard extends React.Component {
```

Comment:
Why'd you convert this to a `class`? I know you're trying to avoid learning hooks at this stage but you've not introduced state and you've not used life cycle methods either  - no reason to use a class!

Reply:
Oh wait I see, you've used context. You can do that with a hook but just stick to what you know for now i guess!

Reply:
Yeah, I'm going to get familiar with the big concepts before I start doing all the funky little stunts that senior devs use. Although I must point out that I actually prefer classes to functions so don't be too surprised to see me use them by default - it means less refactoring later if I need to extend what I have